src/markdown-pages/docs/ecosystem/ecosystem-authentication.md

```diff
@@ -36,7 +36,7 @@ To log out of a Galasa Ecosystem using galasactl, you can use the `galasactl aut
 
 ### Listing personal access tokens
 
-You can retrieve a list of all active personal access tokens in the Ecosystem by using the `galasactl auth tokens get` command. This information is useful if you need to revoke or delete an access token, for example, if a user moves to a new role, or loses a laptop with their access token on it.
+You can retrieve a list of all active personal access tokens in the Ecosystem by using the `galasactl auth tokens get` command. This information is useful if you need to query active tokens with a view to revoking or deleting a particular token, for example, if a user moves to a new role, or loses a laptop with their access token on it.
```

Comment:
Since `auth tokens delete` revokes a personal access token, maybe replace mentions of "deleting" with "revoking" - same applies below